### DISCUSSION TITLE
otel data cannot trigger alarms in cluster mode[Bug]

### DISCUSSION THREAD

**Discussion by ZhangYiJava** [apache/skywalking#10830] [Q&A] [answered]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar issues.

### Apache SkyWalking Component

OAP server (apache/skywalking)

### What happened

In clustered mode, when data is received and deserialized in RemoteServiceHandler, MetricsMetaInfo in AcceptableValue is always initialized (metricsName="UNKONWN",scope=0). As a result, such data cannot trigger alarms in cluster mode

### What you expected to happen

I expect that the metrics monitored by the infrastructure in cluster mode can normally trigger alarms

### How to reproduce

Replication mode: Access infrastructure monitoring data in cluster mode and add alarms to the data

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
The alerting is only based on metrics persistent. If something wrong happened in the `RemoteServiceHandler`, then the metrics in the database are wrong too.
I can't follow up your description.

**Comment by ZhangYiJava** [comment, marked as answer]
![a721ce34c285a5b2138f35d273795b91](https://github.com/apache/skywalking/assets/98068691/7c97afc9-e52d-4a3a-ac9f-908dafb82760)
By this I mean that the MetricsMetaInfo in Meter obtained under RemoteServiceHandler is (metricsName="UNKONWN",scope=0), resulting in no alarm being triggered when the notify method in NotifyHandler is executed.

**Reply by wu-sheng** [reply]
We confirmed this and addressed the root cause. https://github.com/apache/skywalking/pull/13543 will try to fix it.
